client/app/src/accounts/account.controller.js

```diff
@@ -490,16 +486,16 @@
         }
       }, 200)
 
-      self.exchangeSellTransaction = transaction
-      changerService.monitorExchange(resp).then(
+      self.exchangeSellTransaction = transaction // eslint-disable-line no-undef
```

Comment:
These two lines appeared to be the artifact of some refactoring. Wasn't sure if it was appropriate to remove them or not.